tests/lambda_functions/pinpoint_callback/test_pinpoint_callback_lambda.py

```diff
@@ -4,32 +4,23 @@
 from moto import mock_aws
 
 
-@mock_aws
-def create_sqs_queue():
-    sqs = boto3.resource('sqs', region_name='us-gov-west-1')
-    sqs.create_queue(QueueName='vanotify-delivery-status-result-tasks')
-
-
-@pytest.mark.serial
-@mock_aws
 def test_pinpoint_callback_lambda_raises_client_error():
-    # marked serial because this test has to run before the next one so the function is not already in memory
+    # Withuot the SQS data being available to the lambda we expect a ClientError
     with pytest.raises(ClientError):
-        # this is imported here because importing it at the top of the file causes the exception to be thrown there
-        from lambda_functions.pinpoint_callback.pinpoint_callback_lambda import lambda_handler
-
-        lambda_handler({}, {})
+        import lambda_functions.pinpoint_callback.pinpoint_callback_lambda
 
 
 @mock_aws
 def test_pinpoint_callback_lambda_handler_success():
-    # there has to be a better way to do this, right?
-    create_sqs_queue()
-    # this is imported here because the queue must be created before the lambda is imported
+    sqs = boto3.resource('sqs', region_name='us-gov-west-1')
+    sqs.create_queue(QueueName='vanotify-delivery-status-result-tasks')
+    sqs.create_queue(QueueName='test-notification-delivery-status-result-tasks')
+
+    # The import attempts to initialize the SQS resources, has to be imported after the fixture has ran
     from lambda_functions.pinpoint_callback.pinpoint_callback_lambda import lambda_handler
 
     # test data
     event = {'Records': [{'kinesis': {'data': 'test-data'}}]}
     context = {}
 
-    assert lambda_handler(event, context) is None
+    lambda_handler(event, context)
```

Comment:
What does this return now, if not None?

Reply:
If somebody changed the lambda code to return something other than None--possible by accident--should this test still pass?

Reply:
Yes. This is a very simple happy path test.